### DISCUSSION TITLE
Nova add button on detail page

### DISCUSSION THREAD

**Discussion by bhargavhauper** [laravel/nova-issues#3787] [Q&A]
![Screenshot 2022-03-21 at 7 57 30 PM](https://user-images.githubusercontent.com/96163494/159282605-b3b07f47-0a6f-4c3a-8140-8b83285f0b39.png)

This Type Button On specific Task Like Update Status Etc  
Plz Guide me on How To Create this Action Button 

I have used this Package "gobrightspot/nova-detached-actions" But it not working

**Comment by zek** [comment]
That would be a great addition to nova!
Any news?

**Comment by nrueckmann** [comment]
IMO all actions (including default actions like "preview", "edit", "delete", etc.) should behave in the same configurable way. 
So the application can decide which actions should be displayed and how. 
Possible options:
- Icon (this is how current default actions are displayed)
- Text (same as icon but with a string instead of an icon)
- Dropdown (same as current custom actions)
